--- a/configure.ac	2017-12-17 16:52:07.786815291 +0000
+++ b/configure.ac	2017-12-17 16:52:36.794793896 +0000
@@ -212,7 +212,7 @@
   # env -i gives this build host configure a clean environment;
   # consequently, we have to re-initialize $PATH.
   env -i CC="$BUILD_CC" AR="$BUILD_AR" RANLIB="$BUILD_RANLIB" \
-      PATH="$PATH" \
+      PATH="$PATH" LDFLAGS="$BUILD_LDFLAGS" CFLAGS="$BUILD_CFLAGS" \
       tools_only=1 \
     ${confdir}/configure --build=${build} --host=${build} \
                          --disable-rpath --disable-nls
