--- Theano-rel-0.9.0-src/theano/sandbox/cuda/nvcc_compiler.py.orig	2017-06-20 23:29:31.000000000 -0400
+++ Theano-rel-0.9.0-src/theano/sandbox/cuda/nvcc_compiler.py	2017-06-20 23:29:03.000000000 -0400
@@ -101,6 +101,10 @@
         # that use the old API.
         flags.append("-DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION")
 
+        # glibc 2.23 changed memcpy definitions in string.h
+        # https://fossies.org/diffs/glibc/2.22_vs_2.23/string/string.h-diff.html
+        flags.append("-D_FORCE_INLINES")
+
         # If the user didn't specify architecture flags add them
         if not any(['-arch=sm_' in f for f in flags]):
             # We compile cuda_ndarray.cu during import.
